pkg/apis/core/types.go

```diff
@@ -5332,9 +5332,6 @@ type RangeAllocation struct {
 }
 
 const (
-	// DefaultSchedulerName defines the name of default scheduler.
-	DefaultSchedulerName = "default-scheduler"
```

Comment:
We already have `k8s.io/api/core/v1.DefaultSchedulerName`, and I just want to prevent this being referenced by other packages

Reply:
> why drop the constant from this package?

We already have k8s.io/api/core/v1.DefaultSchedulerName, and I just want to prevent this being referenced by other packages